### PR TITLE
feat: remove kam resources from operator

### PR DESCRIPTION
**Pull request by saumeya**
**What type of PR is this?**

 /kind enhancement

**What does this PR do / why we need it**:
To remove kam deployment and routes from the operator code base. Also removing related tests and documentation.

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**Test acceptance criteria**:

* [ ] Unit Test
* [ ] E2E Test

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

**Review by anandf — COMMENTED** [review]
LGTM

File: controllers/.DS_Store
Comment (svghadi) [inline controllers/.DS_Store:1]: Looks like a local os file. Can you remove it from the commit?

**Review by svghadi — COMMENTED** [review]
Left a comment regarding a local file in the PR. Rest all looks good. Thanks.

/remove-hold

**Pull request merged**